src/credentials/sso/diskCache.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import * as crypto from 'crypto'
+import { SsoCache } from './ssoCache'
+import { ClientRegistration } from './clientRegistration'
+import { join } from 'path'
+import { homedir } from 'os'
+import { AccessToken } from './accessToken'
+
+export class DiskCache implements SsoCache {
+    private cacheDir: string = join(homedir(), '.aws', 'sso', 'cache')
+
+    loadClientRegistration(ssoRegion: string): ClientRegistration | null {
```

Comment:
Please explicitly state that functions in this class are `public`

Reply:
Done